<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentFaceWidget</class>
 <widget class="QWidget" name="StudentFaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>太空大学教务管理系统学生端</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="sideBar" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QToolButton" name="examTtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>考试成绩</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/picture/icon/examIcon.png</normaloff>:/picture/icon/examIcon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="personInfoTtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>个人信息</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/picture/icon/personIcon.png</normaloff>:/picture/icon/personIcon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="personPage">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>210</y>
         <width>231</width>
         <height>151</height>
        </rect>
       </property>
       <property name="text">
        <string>个人信息</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="examPage">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>260</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>考试成绩</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
